docs/api/components.md

```diff
@@ -292,6 +292,78 @@ export default connect({
 })
 ```
 
+## Angular 2/4 (BETA)
+[Website](https://angular.io/)
+
+**NPM**
+
+`npm install angular`
+
+```js
+import {NgModule} from '@angular/core'
+import {BrowserModule} from '@angular/platform-browser'
+import {CerebralService} from 'cerebral/angular'
+import {AppComponent}  from './app.component.ts'
+
+const cerebralInitialStore = {
+  state: {
+    myName: 'Momo',
+  }
+}
+
+// AOT support
+export function startCerebral() {
+  return new CerebralService(cerebralInitialStore)
+}
+
+@NgModule({
+  imports:      [ BrowserModule ],
+  declarations: [ AppComponent ],
+  bootstrap:    [ AppComponent ],
+  providers: [
+     {
+      provide: CerebralService,
+      useFactory: startCerebral
+    },
+  ]
+})
+export class AppModule {}
+```
+
+```js
+import {Component, ChangeDetectorRef} from '@angular/core'
+import {state, signal} from 'cerebral/tags'
+import {Cerebral} from 'cerebral/angular'
+import {CerebralService} from 'cerebral/angular'
+
+@Component({ 
+  selector: 'app-component',
+  template: `
+    <div (click)="onClick()">{{foo}}</div>
+  `,
+  changeDetection: ChangeDetectionStrategy.OnPush // disable angular change detection
+})
+@Cerebral({
+  myName: state(['foo']),
+  onClick: signal(['mySignal'])
```

Comment:
What is the reason to have an exceptional way of using tags for this API ? Why not use tagged templates ?

```js
myName: state`foo`,
onClick: signal`mySignal`,
```